<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FromFile</class>
 <widget class="QWidget" name="FromFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="button_select">
     <property name="text">
      <string>Select audio file</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QWebEngineView" name="webEngineView_tabs">
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="button_generate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Generate tabs</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="audio_name">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="button_return">
     <property name="text">
      <string>Return</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
